src/d3a/models/strategy/utils.py

```diff
@@ -0,0 +1,40 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU
+General Public License as published by the Free Software Foundation, either version 3 of the
+License, or(at your option) any later version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with this program. If not,
+see <http://www.gnu.org/licenses/>.
+"""
+import numpy as np
```

Comment:
Done.

Reply:
I've checked the run times with and without numpy to be sure. If you're interested:
```python
from timeit import timeit


number_of_runs = 1000000
energy_kWh = 1000
std = (50 * energy_kWh) / 100

python_gauss = timeit(
    "random.gauss(mu=energy_kWh, sigma=std)",
    setup=f"import random; energy_kWh={energy_kWh}; std={std}",
    number=number_of_runs
)
numpy_normal = timeit(
    "np.random.normal(loc=energy_kWh, scale=std)",
    setup=f"import numpy as np; energy_kWh={energy_kWh}; std={std}",
    number=number_of_runs
)

print(f"Python random.gauss:\t{python_gauss}s")
print(f"NumPy random.normal:  \t{numpy_normal}s")

# Python random.gauss:	0.5805347780000005s
# NumPy random.normal:  	2.389011778000011s
```

Reply:
Hmm, we already install numpy and for a good reason, to not fall into the rabbit hole that we had once when we were fixing the random seed. Is using numpy really using more resources than the `random.gauss()` ?
But more importantly: Is `random.gauss()` always creating the same random numbers, I remind you on our long discussion [here](https://github.com/gridsingularity/d3a/pull/1196#discussion_r681732859)

Reply:
Sorry, I did not see your last comment, we must have written them in parrelel.

Reply:
@hannesdiedrich in order not to alter the global seed, I'm using `random.Random(12)`. This is the same I was doing earlier with numpy: we create a new instance of a random number generator and we alter the seed only on this instance (not on the global `random` module). So it should have the same effect as before and it will always generate the same random numbers.

Reply:
Ok, great! Thank you!